### PR TITLE
PUD-1193 - remove serenity

### PR DESCRIPTION
**Pull request by jonnywyatt**

### CODE REVIEW

**Comment by dazoakley** [comment]
So we're totally not bothering with the local checks now?

**Comment by jonnywyatt** [comment]
@dazoakley I spent all of Friday on it and was unable to work out why it's failing. Rather than putting more time into it at this point, I think it's better to disable it and either come back to it or work out a simpler way to test against mocks.

**Comment by dazoakley** [comment]
As long as the others are happy with this all good with me 👍

File: README.md [hunk 1]
```diff
@@ -64,9 +33,9 @@ There are multiple options for running these tests from your local:
 In this scenario the login credentials must be valid on the target environment. Hence, your env vars
 needs to reflect the same.  See above. Also, need to add a valid noms number if running against preprod/prod. Use `NOMS_NUMBER=X1234YZ`.
 
-Given valid credentials, to run against one of our deployed environments use simply:
+Given valid credentials, to run against dev environment use:
 ```
-./gradlew test -Denvironment={dev/preprod/prod}
+npx cypress open --env NOMS_NUMBER=A7826DY,USERNAME=[USER],PASSWORD=[PASSWORD] --config baseUrl=https://manage-recalls-dev.hmpps.service.justice.gov.uk
```
Comment (philsegal) [inline README.md:38]: Is this just a test noms number?

File: .circleci/config.yml [hunk 2]
```diff
@@ -103,16 +89,14 @@ jobs:
...
           paths:
             - ~/.npm
             - ~/.cache
       - run:
           name: "Run Cypress tests [<<pipeline.parameters.e2e_environment>>]"
           command: |
             set +e
-            npm i
-            npx cypress run --env USERNAME=${CYPRESS_USERNAME_<<pipeline.parameters.e2e_environment>>},PASSWORD=${CYPRESS_PASSWORD_<<pipeline.parameters.e2e_environment>>},NOMS_NUMBER=${NOMS_NUMBER_<<pipeline.parameters.e2e_environment>>} --config baseUrl=https://manage-recalls-<<pipeline.parameters.e2e_environment>>.hmpps.service.justice.gov.uk
+            npx cypress run --env USERNAME=${CYPRESS_USERNAME_<<pipeline.parameters.e2e_environment>>},PASSWORD=${CYPRESS_PASSWORD_<<pipeline.parameters.e2e_environment>>},NOMS_NUMBER=${NOMS_NUMBER_<<pipeline.parameters.e2e_environment>>} --config baseUrl=https://manage-recalls-<<pipeline.parameters.e2e_environment>>.hmpps.service.justice.gov.uk --browser chrome
```
Comment (philsegal) [inline .circleci/config.yml:99]: Once this is merged, can you remove SERENITY_USERNAME_* and PASSWORD_* from circle?

**Review by philsegal — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jonnywyatt) [reply README.md:38]: yes, an example that works against dev. I'll add a note to swap in one that works for the env it's being run against

File: .circleci/config.yml (hunk 2 above)
Comment (jonnywyatt) [reply .circleci/config.yml:99]: yep

**Review by philsegal — APPROVED** [review]
Looks great! Brilliant step forwards!

**Pull request merged**